apps/cookbook/src/app/examples/grid-layout-example/grid-layout-single-container-example/grid-layout-single-container-example.component.html

```diff
@@ -0,0 +1,75 @@
+<main>
+  <div class="grid-container">
+    <div class="grid-item">
+      <h1>Example Content 1</h1>
```

Comment:
Consider changing to the following for clarity, or add another heading above it.
```suggestion
      <h1>Grid Layout - single grid container</h1>
```